docs/paper/paper.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'SWMManywhere: Synthesise Urban Drainage Network Models Anywhere in the World'
+tags:
+  - python
+  - stormwater
+  - hydrology-stormwater-analysis
+  - swmm5
+  - swmm
+  - hydraulic-modelling
+authors:
+  - name: "Barnaby Dobson"
+    orcid: "https://orcid.org/0000-0002-0149-4124"
+    affiliation: 1
+  - name: "Diego Alonso-Álvarez"
+    orcid: "https://orcid.org/0000-0002-0060-9495"
+    affiliation: 1
+  - name: "Taher Chegini"
+    orcid: "https://orcid.org/0000-0002-5430-6000"
+    affiliation: 2
+affiliations:
+ - name: Imperial College London, UK
+   index: 1
+ - name: Purdue University, US
+   index: 2
+date: 16 October 2024
+bibliography: paper.bib
+---
+
+## Summary
+
+Urban drainage network models (UDMs) are useful for a variety of purposes, most notably simulating and alleviating pluvial flooding. Despite the clear usefulness of UDMs, they are often not used owing to the difficulty and expense of creating them. This creates a significant gap for users attempting to generate UDMs if they are not able to perform an expensive underground survey. A variety of literature exists to derive such UDMs from GIS data, producing hydraulically feasible models that closely approximate real-world systems [@Blumensaat2012-hd;@Chahinian2019-lg;@Reyes-Silva2022-pr]. We identify some key limitations of these approaches, most notably the lack of automatic data acquisition and preprocessing, that all approaches are closed-source to date, and that a key feature of such an approach should be to facilitate extension and customisation.
```

Comment:
Please add my paper to the citations that you listed too, since that's what it does.